pages/linux/clamav.md

```diff
@@ -0,0 +1,21 @@
+# clamav
```

Comment:
Could we rename this page to `clamscan` please? tldr pages should be named after the command they document.